packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -212,4 +214,12 @@ Will double processing times. Use only for debugging purposes.",
     default: defaultOptions.chain.maxShufflingCacheEpochs,
     group: "chain",
   },
+
+  "chain.archiveBlobs": {
+    description:
+      "Number of epochs to retain the finalized blobs (min MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS), set to infinity (retain all) with just --chain.archiveBlobs",
```

Comment:
Would prefer a simpler cli flag:
- no special handling for Infinity (User can just do `--option Infinity`)
- call it chain.archiveBlobEpochs

```suggestion
  "chain.archiveBlobEpochs": {
    description:
      "Number of epochs to retain finalized blobs (minimum of MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS)",
```